lib/rules/web/admin_console/4.4/yaml_page.xyaml

```diff
@@ -0,0 +1,62 @@
+click_yaml_tab:
+  params:
+    tab_name: YAML
+  action: click_tab
+open_view_sidebar:
+  params:
+    button_text: View sidebar
+  action: click_button
+  element:
+    selector:
+      xpath: //button[contains(.,'Schema')]
+close_breadcrumb_sidebar:
```

Comment:
`close_breadcrumb_sidebar` is a little confusing , how about `close_view_sidebar`